lib/Support/ProgramCounterHandler.cpp

```diff
@@ -109,24 +122,11 @@ class ARMProgramCounterHandler : public ProgramCounterHandler {
     revng_assert(affectsPC(Store));
 
     Value *Pointer = Store->getPointerOperand();
-    Value *StoredValue = Store->getValueOperand();
 
     if (Pointer == IsThumb) {
-      // Update Type
-      using CI = ConstantInt;
-      using namespace MetaAddressType;
-
-      auto *TypeType = getCSVType(TypeCSV);
-      auto *ArmCode = CI::get(TypeType, Code_arm);
-      auto *ThumbCode = CI::get(TypeType, Code_arm_thumb);
-      // We don't use select here, SCEV can't handle it
-      // NewType = ARM + IsThumb * (Thumb - ARM)
-      auto *NewType = B.CreateAdd(ArmCode,
-                                  B.CreateMul(B.CreateTrunc(StoredValue,
-                                                            TypeType),
-                                              B.CreateSub(ThumbCode, ArmCode)));
-
-      B.CreateStore(NewType, TypeCSV);
+      // Compute Type and update it.
```

Comment:
* [ ] What's "it"?
* [ ] No dots at the end of comments

Reply:
1) The comment before was `Update Type`, then I changed it to `Compute Type and update it`, but now that I think about it, the original one was fitting better.
2) There are comments with ending dots as many as with not in the codebase. For example when introducing context/background, the final dot makes sense (after like 4/5 lines of comments). So, can I just remove it here or do I need to remove the final dot everywhere else?

Reply:
If it's a single sentence, no dot.

Reply:
OK, got it, closing this as I'm removing all the dots from single sentences that I've inserted.